Vagrantfile

```diff
@@ -132,6 +132,13 @@ provision_node = <<SCRIPT
  -listen-peer-urls http://#{node_addr}:2380 \
  -initial-cluster #{node_peers} \
  -initial-cluster-state new 0<&- &>/tmp/etcd.log &) || exit 1
+
+if [ -f "#{netplugin_synced_gopath}/bin/docker" ]
```

Comment:
Will it be better if we pass the path to docker binary as a separate environment variable say CONTIV_DOCKER_BIN? 

It will give more flexibility to try any docker binary from the host and also help in cases where docker binary might not be there in gopath, for instance for me somehow the docker binary just gets generated in the `bundles/<docker-release>` directory in my docker workspace and doesn't get's placed in my `$GOPATH/bin` directory.

Also slightly related, there may be a use case to have this facility for other relevant binaries as well (like rkt, k8s etc), not sure if there is an easy generic way to address all. But I am ok to take care of that separately.